### PR TITLE
feat(macros): supports customizing v-model in Vue 2

### PR DESCRIPTION
**Pull request by duowb**
### Description

add vue2 [Customizing-Component-v-model](https://v2.vuejs.org/v2/guide/components-custom-events.html#Customizing-Component-v-model)

[Example](https://codesandbox.io/s/red-sea-ycblru)

### Additional context
How to test `defineOptions` and `defineModel` together? 😢

### CODE REVIEW

**Comment by sxzz** [comment]
You can add new test file to `packages/macros/tests/fixtures`, like `mixed.vue`.

**Comment by sxzz** [comment]
I renamed some variables in your PR, please don't forget to pull the code before new commits.

File: packages/macros/tests/fixtures/define-model/vue2-custom-vmodel.vue [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<script lang="ts">
+export default {
```
Comment (sxzz) [inline packages/macros/tests/fixtures/define-model/vue2-custom-vmodel.vue:2]: Maybe we should handle another situation: export default with `defineComponent` function.

```ts
export default defineComponent({
  model: {
    prop: 'value',
    event: 'changeInput'
  },
})
```

**Comment by duowb** [comment]
image

`packages\macros\package.json` Is it necessary to delete

**Comment by sxzz** [comment]
@duowb No, `macros` package also need provide `defineOptions` feature.

**Comment by duowb** [comment]
> @duowb不，`macros`包还需要提供`defineOptions`功能。
run `ni`
 ERR_PNPM_NO_MATCHING_VERSION_INSIDE_WORKSPACE  In packages\macros: No matching version found for unplugin-vue-define-options@* inside the workspace

**Pull request merged**